### PR TITLE
Remove non-emitted PostAsyncInsertObjectsEvent

### PR DESCRIPTION
**Pull request by bobvandevijver**
I believe this event is not emitted anywhere from the source, so it should be save to remove.

### CODE REVIEW

**Comment by bobvandevijver** [comment]
@XWB Any update on this MR?

**Comment by XWB** [comment]
We don't know if someone uses the event, removing it would be a BC break. Instead, you should deprecate the event so it can be removed in a future major release.

**Comment by bobvandevijver** [comment]
Fair enough, updated!

File: src/Command/PopulateCommand.php [hunk 1]
```diff
@@ -171,13 +170,6 @@ private function populateIndex(OutputInterface $output, string $index, bool $res
             }
         );
 
-        $this->dispatcher->addListener(
```
Comment (XWB) [inline src/Command/PopulateCommand.php:174]: I think this change needs to be reverted, otherwise custom event listeners won't be triggered.

Comment (bobvandevijver) [reply src/Command/PopulateCommand.php:174]: I'm not sure what you mean here. This command registers its own listener for an event (that is never emitted at all). This listener registration does not create any event by itself?

**Review by XWB — APPROVED** [review]

**Comment by XWB** [comment]
Approved, but there is a merge conflict.

**Comment by bobvandevijver** [comment]
@XWB Merge conflict has been fixed!

**Comment by XWB** [comment]
Thanks :)

**Pull request merged**